### ISSUE TITLE
Feature-Request: Suchparameterabbildung in der URL

### ISSUE THREAD

**Issue by Olli** [heimrichhannot/contao-filter-bundle#12]
Kann man feste Suchen als URL darstellen?
z.B. irgendwas wie /filter/fuechte/äpfel/farbe/rot oder so um z.B. damit verlinkbare Unterseiten aus der Suche zu generieren.

**Comment by Defcon0** [comment]
Ja, das geht. Du kannst dafür das Inhaltselement "Filter-Vorauswahl" nutzen. Leg einfach eine Seite unter der gewünschten URL an und lege dort das Inhaltselement rein. In diesem kannst du die gewünschten Filter setzen. Allerdings muss der Filter die entsprechenden FIlter-Elemente bereits haben und du kannst sie dort dann überschreiben.

**Comment by Olli** [comment]
Ja das kann man so machen. Ich meinte das aber generell. Wenn die URL auch per JS angepasst werden kann, kann man per Vor-Zurück im Browser navigieren und man kann z.B. in Artikeln schnell und einfach Suchen verlinken.

**Comment by Defcon0** [comment]
In der Form geht es noch nicht.

**Comment by Olli** [comment]
Wäre super wenn das als Feature mal in Betracht gezogen werden könnte :+1:
